src/api/content-rest-api/model/pagination.ts

```diff
@@ -15,6 +15,18 @@
 * limitations under the License.
 */
 
+export interface PaginatedList<T> {
```

Comment:
For what is a PaginatedList?

Reply:
It describes the payloads for some APIs (see custom models client)